src/App.js

```diff
@@ -1,13 +1,38 @@
 import React, { Component } from 'react';
 import './App.css';
 import Header from './Header.js';
+import pokeData from './data.js'
+import ImageList from './ImageList.js'
+import request from 'superagent';
 
 export default class App extends Component {
   
+state = {
+  pokeApi: [],
+}
+
+
+async componentDidMount() {
+  const pokeApiData = await request.get(`https://alchemy-pokedex.herokuapp.com/api/pokedex`);
+
+  const pokeApi = this.state.pokeApi;
```

Comment:
you could drop this pokeApi variable because it seems as if it isnt doing anything.